daisy_workflows/ovf_import/create_gmi.wf.json

```diff
@@ -0,0 +1,98 @@
+{
+  "Name": "create-instance",
+  "DefaultTimeout": "240m",
```

Comment:
Hmm I don't remember what's the original number, but I feel I don't see 240m anywhere before. Is it align with the original design?

Reply:
Good question -- it's the value that's used in the current OVF workflows, eg: 

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/22dbfbdebb2353295698e93c2089ded45c56707b/daisy_workflows/ovf_import/import_ovf_to_instance.wf.json#L3